uint8 mode

uint8 READY = 0
uint8 END = 10
uint8 HOLD = 11
uint8 BLUE_PURSUIT = 12
uint8 LURE_DEEP = 13
